frontend/src/store/modules/organization.js

```diff
@@ -15,6 +15,23 @@ const state = {
 };
 
 const getters = {
+    ancestorsByName: (state) => (name) => {
+        let ancestors = [];
+        for (let topOrg in state.orgList) {
+            if (state.orgList[topOrg].name == name) {
+                return ancestors;
+            }
+            let children = state.orgList[topOrg].children;
+            for (let child in children) {
+                if (children[child].name == name) {
+                    ancestors.push(state.orgList[topOrg].name);
+                    return ancestors;
+                }
+            }
+        }
+        return ancestors;
+    },
```

Comment:
I agree the use of ancestors is confusing as currently we only do one level of checking instead of recursive checking. I'll remove the use of ancestors. Will leave the rest of the style the same as it matches the other functions in the file.